tests/core/dom/element/isdetached.js

```diff
@@ -0,0 +1,88 @@
+/* bender-tags: editor,dom */
```

Comment:
There is no test for detached document.

Reply:
I add test for current document, for new detached document, and some element inside detached document.